doc/release-notes.md

```diff
@@ -1,3 +1,56 @@
+# Release 0.14
```

Comment:
I wasn't sure if this was the right section or if it should be in the 13 section or a new 13.x section.

Reply:
I think it should probably go in a 0.13.3 section

Reply:
Done.